### DISCUSSION TITLE
【求助】绳网上可不可以发视频

### DISCUSSION THREAD

**Discussion by hzjwork** [share121/inter-knot#626] [其他]
问下各位绳网上可不可以发视频？能发的话最大可以发多大的视频？

**Comment by FrostPhoenix79** [comment]
试了一下，文件超过10MB就不让我发了。或许广大绳匠可以做到，我赌10菲林。

**Comment by capacitor1** [comment]
可以。GitHub内可以10MB以下的

但是可以引用外部链接，可以把视频发文件代理站上在利用markdown把链接引用一下
